src/cpp/shared/game/game.cpp

```diff
@@ -382,12 +382,12 @@ void game::ProcessOnMapHeroes() {
         }
 
         if (!isJail) {
-          this->SetVisibility(randomHero->x, randomHero->y, randomHero->ownerIdx, giVisRange[randomHero->secondarySkillLevel[3]]);
+          this->SetVisibility(randomHero->x, randomHero->y, randomHero->ownerIdx, giVisRange[randomHero->secondarySkillLevel[3]]); // probably SECONDARY_SKILL_SCOUTING
```

Comment:
I mean, replace that 3 with this: https://github.com/jkoppel/project-ironfist/blob/master/src/cpp/shared/adventure/adv.h#L30

Reply:
I accidentally forgot to change it.